.storybook/static/preview-head.js

```diff
@@ -375,3 +375,44 @@ function toggleNestedListSubmenu(event) {
         icon.classList = 'sap-icon--navigation-down-arrow';
     }
 }
+
+function collectionHas(a, b) { //helper function (see below)
+    for(var i = 0, len = a.length; i < len; i ++) {
+        if(a[i] == b) return true;
+    }
+    return false;
+}
+
+function toggleCondensedVerticalNavSubmenu(event) {
+    let button = event.target;
+
+    while (button.id !== 'parentCalendarButton') {
+        button = button.parentNode;
+    }
+
+    if(!button.classList.contains('is-expanded')) {
+        button.classList.add('is-expanded');
+    } else if(button.classList.contains('is-expanded')) {
```

Comment:
I think `else if` isn't necessary here, same result could be achieved the following way:
```
if(button.classList.contains('is-expanded')) {
    button.classList.remove('is-expanded');
} else {
    button.classList.add('is-expanded');
}
```